pages/common/gh.md

```diff
@@ -0,0 +1,32 @@
+# gh
+
+> Work seamlessly with GitHub from the command line.
+> More information: <https://cli.github.com/>.
+
+- Create a pull request:
+
+`gh pr create`
+
+- View a pull request in the browser:
+
+`gh pr view {{1}}`
```

Comment:
Sorry to be a pain, but perhaps this would read better with a word instead. Like this?
```suggestion
`gh pr view {{pr_number}}`
```

Same goes for the other examples.

Reply:
No worries, any feedback is welcome for a newcomer in this repo